test/run-openshift

```diff
@@ -169,9 +169,7 @@ function test_postgresql_update() {
     esac
   done
 
-  if docker pull "$old_image" 2>/dev/null; then
-    $released
```

Comment:
Hm, if there's no other option ... but I added the $released check on purpose, to check that released variable is set to valid value (aka that NOT_RELEASED_VERSIONS is not staled)

Reply:
I see, makes sense, thanks!
In that case I can add something like `$released || [ "$OS" = "centos7" ]` to make the exception centos specific (as we release centos images directly after a PR is merged)